processing/src/main/java/org/apache/druid/segment/DimensionDictionary.java

```diff
@@ -96,6 +99,12 @@ public int size()
     }
   }
 
+  public long sizeInBytes()
+  {
+    // TODO: size of map/list object itself?
```

Comment:
Make sure to do something to remove the TODOs before merge.  Either implement or add a comment that explains what's missing and why it's potentially an issue or just delete.